credentialMixin.js

```diff
@@ -27,16 +27,19 @@ export const credentialMixin = {
     return {
       showFieldValues: false,
       useDefaultImage: false,
+      svg: false
     };
   },
   created() {
     // Defaults shown fields to 3 if not defined as prop
     if(this.fieldQuantity === undefined) {
       this.fieldQuantity = 3;
     }
-    if(this.credential.issuerLogo === undefined) {
-      this.credential.issuerLogo = this.defaultImage;
+    // Checks if issuerLogo is SVG string
+    if(this.credential.issuerLogo.includes('<svg')) {
```

Comment:
This check is brittle. It will pass for any element that contains an svg. We want it to only pass if the element passed in is an svg.

Both cases below would pass with your check.

**Good**
```html
<svg>
  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
</svg>
```

**Bad**
```html
<div>
  <div>
    <h1>This should not pass</h1>
  </div>
  <div>
     <svg>
      <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
    </svg>
  </div>
</div>
```

Additionally, why did you not use a computed property here?

Reply:
I think there's a better solution for this. Let's remove all of the svg logic altogether. The component is starting to take on more responsibility than it should.

This component is responsible for displaying credentials and that's it. Complex image logic is external to it. At this level, the component should get a well-formatted image and display it. Preferably a URL. If the user of the API can't provide a URL (our current situation) then we should just use a slot for the image.

The change should be to remove the svg code. Create a slot for a user-supplied image. If the slot is used then display that image, else fallback to the default image.